### DISCUSSION TITLE
Different configs in local and production environment

### DISCUSSION THREAD

**Discussion by jcerdan** [BePsvPT/secure-headers#87] [Q&A] [answered]
Hello,

I'm using this package for CSP settings in a website, but I have a problem with local and production environments. 
Both share the same config file but I would like to have different config files as production environment would have stronger policies (not allowing http: scheme for example)
I don't really know how to do that as App Object is not available when reading the config php file.

Has anyone met and solved this problem?

Best

**Comment by bepsvpt** [comment, marked as answer]
Hello @jcerdan,

You can dynamically adjust the secure-headers config in `AppServiceProvider`.

**Reply by jcerdan** [reply]
Hi,

Thanks @bepsvpt 

Good. I'll try with mergeConfigFrom() in AppServiceProvider to load the production config file in production env.

Regards,

**Comment by incoming-th** [comment]
@jcerdan Maybe this could help, I had similar requirement and I wanted to have everything in 1 single config file so I ended up doing this in the secure-headers.php file:

```
'unsafe-inline' => app()->runningInConsole() === false ? env('APP_ENV') === 'local' : false,
```

You will need to use "app()->runningInConsole()" otherwise it will fail during deployment.
